tests/src/test/java/alluxio/IntegrationTestUtilsTest.java

```diff
@@ -0,0 +1,128 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio;
+
+import alluxio.underfs.UnderFileSystem;
+import alluxio.underfs.gcs.GCSUnderFileSystem;
+import alluxio.underfs.hdfs.HdfsUnderFileSystem;
+import alluxio.underfs.local.LocalUnderFileSystem;
+import alluxio.underfs.oss.OSSUnderFileSystem;
+import alluxio.underfs.s3.S3UnderFileSystem;
+import alluxio.underfs.s3a.S3AUnderFileSystem;
+import alluxio.underfs.swift.SwiftUnderFileSystem;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+/**
+ * Tests for {@link IntegrationTestUtils}.
+ */
+public class IntegrationTestUtilsTest {
+  @Test
+  public void gcs() {
+    UnderFileSystem underFileSystem = Mockito.mock(GCSUnderFileSystem.class);
+    Mockito.when(underFileSystem.getUnderFSType()).thenCallRealMethod();
+
+    Assert.assertTrue(IntegrationTestUtils.isGcs(underFileSystem));
```

Comment:
I wonder if there is a way to use iteration based on a `HashMap` to generate these tests so that it is easier to add tests for a new under storage